media-video/pipewire/pipewire-9999.ebuild

```diff
@@ -41,6 +41,7 @@ BDEPEND="
 	)
 "
 RDEPEND="
+	acct-group/pipewire-desktop
```

Comment:
Like discussed in IRC, I'll probably prefer using `audio` group which is already in use by everyone following handbook. No final decision yet.

Reply:
As I explained multiple times (and as you verified in your own tests), audio group should not be used at all for non-system daemons (which we do not support on Gentoo!). All it does is interfere with Access Control List maintained by either elogind or systemd-logind causing unpleasant experiences such as no access to audio devices when another logged in user has taken them, including that users audio being played back even though they have lost the active seat.